index.css

```diff
@@ -82,32 +96,45 @@ header li a:hover {
   color: #cfcfcf;
 }
 
+.page4 .mediacontainer ul li i {
+  margin: 10px;
+  color: #505f79;
+}
+
 .asidedesktop ul li a:hover i {
   color: rgb(248, 176, 245);
 }
 
 .home {
-  /* border: 1px solid white; */
-  position: absolute;
-  display: flex;
-  top: 30vh;
-  width: 85%;
-  left: 15%;
+  /* border: 1px solid red; */
+  grid-column: 2;
+  grid-row: 2 / 3;
 }
 
 .home .text {
-  flex: 1;
+  /* border: 1px double blue; */
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
- [ ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  flex-direction: column;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉